scipy/integrate/_quad_vec.py

```diff
@@ -196,6 +196,21 @@ def quad_vec(f, a, b, epsabs=1e-200, epsrel=1e-8, norm='2', cache_size=100e6, li
     ----------
     [1] R. Piessens, E. de Doncker, QUADPACK (1983).
 
+    Examples
+    --------
+    We can compute integrations of a vector-valued function:
+
+    >>> from scipy.integrate import quad_vec
+    >>> import matplotlib.pyplot as plt
+    >>> alpha = np.linspace(0.0, 10.0, num=30)
+    >>> f = lambda x: x**alpha
+    >>> x0, x1 = 0, 2
```

Comment:
How about making it to range from 0 to 2, because then from the plot you can maybe see that the result is `1/(alpha + 1)`, whereas with `x1=2` the analytical result is harder to understand.

Reply:
Good idea. I changed alpha range from 0 to 2.